### DISCUSSION TITLE
How to enable versioning?

### DISCUSSION THREAD

**Discussion by Mirgos** [pimcore/pimcore#9168] [Q&A / Support]
Hello,

I have a problem with versioning enabling. In var/config/system.yml I have:

```objects:
        versions:
            steps: 5
            days: null 
```
But in pimcore System Settings it looks like this:

![image](https://user-images.githubusercontent.com/18301531/118946731-f98c9c80-b956-11eb-8d41-f650dd3d09dc.png)

Ofc versioning is not working. How to enable this? 

Thanks for help.

**Comment by dvesh3** [comment]
Pimcore version please? it works for me on Pimcore X
![image](https://user-images.githubusercontent.com/5137917/118951638-8c2f3a80-b95b-11eb-8e03-9e94d4bc7e68.png)

**Reply by Mirgos** [reply]
6.8
